consensus/common_test.go

```diff
@@ -291,11 +298,20 @@ func simpleConsensusState(nValidators int) (*ConsensusState, []*validatorStub) {
 	cs := NewConsensusState(state, proxyAppCtxCon, blockStore, mempool)
 	cs.SetPrivValidator(privVals[0])
 
+	// from the updateToState in NewConsensusState
+	<-cs.NewStepCh()
+
 	evsw := events.NewEventSwitch()
 	cs.SetFireable(evsw)
+	evsw.OnStart()
```

Comment:
This is from earlier, but generally we should call Start() rather than OnStart().  Start() first checks to see that Start() wasn't already called... OnStart() is what Service calls when it determines that the services should actually be started.